react/features/cropPerson/functions.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import { getJitsiMeetGlobalNS, loadScript } from '../base/util';
+
+/**
+ * Returns promise that resolves with the crop person effect instance.
+ *
+ * @returns {Promise<JitsiStreamCropPersonEffect>} - Resolves with the crop person effect instance.
+ */
+export function getCropPersonEffect(stream) {
+    const ns = getJitsiMeetGlobalNS();
+
+    if (ns.effects && ns.effects.createCropPersonEffect) {
+        return ns.effects.createCropPersonEffect(stream);
+    }
+
+    return loadScript('libs/video-blur-effect.min.js').then(() => ns.effects.createCropPersonEffect(stream));
```

Comment:
weren't you going to rename this file?

Reply:
Yes, renamed the bundle to tensorflow-effects after checking with Aaron.